taskt/Core/Automation/Commands/DataTable Commands/WriteDataRowValueCommand.cs

```diff
@@ -1,68 +1,70 @@
-        [Attributes.PropertyAttributes.InputSpecification("Enter a existing DataTable to add rows to.")]
-        [Attributes.PropertyAttributes.SampleUsage("**myData**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_DataRowName { get; set; }
+        [PropertyDescription("DataRow")]
+        [InputSpecification("Enter a existing DataRow to add values to.")]
+        [SampleUsage("{vDataRow}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public string v_DataRow { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Select value by Index or Column Name")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Index")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Column Name")]
-        [Attributes.PropertyAttributes.InputSpecification("Select whether the DataRow value should be found by index or column name")]
-        [Attributes.PropertyAttributes.SampleUsage("Select from **Index** or **Column Name**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Search Option")]
+        [PropertyUISelectionOption("Column Name")]
+        [PropertyUISelectionOption("Index")]
+        [InputSpecification("Select whether the DataRow value should be found by index or column name.")]
+        [SampleUsage("")]
+        [Remarks("")]
         public string v_Option { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please enter the index of the DataRow Value")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter a valid DataRow index value")]
-        [Attributes.PropertyAttributes.SampleUsage("0 or [vIndex]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Search Value")]
+        [InputSpecification("Enter a valid DataRow index or column name.")]
+        [SampleUsage("0 || {vIndex} || Column1 || {vColumnName}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_DataValueIndex { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please enter the Value")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the value to write to the DataRow cell")]
-        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Cell Value")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
```

Comment:
Move this **UIHelper** next to Remarks()

Reply:
done